gonzalespaulb/Pokemon-App
# Jd 84 searchbar

## Changes
1. Added style on typeahead searchbar for better UI. 
2. Added style on typeahead scrollbar.

## Purpose
User should be able to easily search for a pokemon and render to the sidebar. 

## Approach
Styled the typeahead extension using scss and make sure to have all pokemons using the searchbar. 

## Testing Steps
1. Search for pokemons using the searchbar and make sure the selected pokemon render on sidebar. 
2. Test the scrollbar and make sure to have access to all pokemons. 


Closes #84
